nodejs16.x/response-streaming/{{cookiecutter.project_name}}/buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
```

Comment:
where is this buildspec used? It doesn't seem CodeBuild is mentioned anywhere.

Reply:
Should we just remove the buildspec? 
My reason is this doesn't seem to belong to here. If customers want to setup CI/CD, they can use `sam pipeline init` after initializing the project with `sam init`

Reply:
I added a comment in the README. Let me know if that's enough or we want a more detailed explanation.

Reply:
I would recommend to leave it to customer's decision to select their CI/CD environment. And they can utilize `sam pipeline` command as @hawflau mentioned above.

Reply:
For some reason my GitHub page didn't load the second comment from Wing yesterday so I didn't see it. I guess I'll remove the buildspec file then.